### ISSUE TITLE
[AVM Module Issue]: retention_period_days vs retention_period_in_days in the backup block 

### ISSUE THREAD

**Issue by vaiddas** [Azure/terraform-azurerm-avm-res-web-site#74]
### Check for previous/existing GitHub issues

- [X] I have checked for previous/existing GitHub issues

### Issue Type?

Bug

### (Optional) Module Version

0.6.0

### Description

under https://github.com/Azure/terraform-azurerm-avm-res-web-site/blob/20e78a8c072e32686e246be7ed1fa495dadc80ad/variables.tf#L504

we have backup variable defined and under schedule there is parameter called "retention_period_in_days"

while the main.function_app.tf is expecting

https://github.com/Azure/terraform-azurerm-avm-res-web-site/blob/20e78a8c072e32686e246be7ed1fa495dadc80ad/main.function_app.tf#L784C11-L784C74

**retention_period_days    = schedule.value.retention_period_days**

due to this backup block throws an error. 

```
│   on main.web_app.tf line 922, in resource "azurerm_linux_web_app" "this":
│  922:           retention_period_days    = schedule.value.retention_period_days
│     ├────────────────
│     │ schedule.value is object with 5 attributes
│
│ This object does not have an attribute named "retention_period_days".
```

**Comment by donovm4** [comment]
Hi @vaiddas - will look into this shortly

**Comment by donovm4** [comment]
addressed in v0.6.2 release
